licenses/print.py

```diff
@@ -0,0 +1,143 @@
+from .models import LicenseRequest
+from PyPDF2 import PdfReader
+from PyPDF2 import PdfWriter
+from datetime import date
+from django.conf import settings
+from django.http import FileResponse
+from django.utils.translation import gettext as _
+from reportlab.pdfgen import canvas
+import io
+
+
+def generate_license_file(user, lr: LicenseRequest) -> FileResponse:
```

Comment:
Is there a way to actually fill the fields in the form? I found one [tutorial](https://medium.com/@vivsvaan/filling-editable-pdf-in-python-76712c3ce99) but it needs [pdfrw](https://pypi.org/project/pdfrw2/). But I find it more helpful, to have it directly in the fields. On the other hand, we have the feature of non-editable data on that license request, so we are sure, it differs not from the database content.

Reply:
https://projects.gocept.com/issues/15901